HACKING.md

```diff
@@ -24,7 +24,7 @@ To build this project you must first install several command line utilities and
 - [`make`](https://www.gnu.org/software/make/) - Make build system
 - [`mvn`](https://maven.apache.org/index.html) (version 3.5 and above) - Maven CLI
 - [`helm`](https://helm.sh/) - Helm Package Management System for Kubernetes
-    - After installing Helm be sure to run `helm init`.
+    - If you are using any Helm 2+ (exluding Helm 3+ versions), after installing Helm be sure to run `helm init`.
```

Comment:
I would recommend something slightly less confusing to a reader, easier as well to pick this out if we ever deprecate the helm 2 charts
```
- If you are using the Helm version 2 charts, after installing the Helm CLI ensure you run `helm init` to configure to your cluster.
```

Reply:
Sure, will do.